cloudwatch.tf

```
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018-2019 Expedia Inc.
+ * Copyright (C) 2018-2020 Expedia Inc.
```

```suggestion
 * Copyright (C) 2018-2020 Expedia, Inc.
```
(yes, the comma is apparently important according to our legal team)

---

Done everywhere